analysis/aws_rules_s3_access_logs/aws_s3_unauthenticated_access.yml

```diff
@@ -1,6 +1,6 @@
 AnalysisType: rule
 Filename: aws_s3_unauthenticated_access.py
-PolicyID: AWS.S3ServerAccess.UnauthenticatedAccess
+PolicyID: AWS.S3.ServerAccess.Unauthenticated
```

Comment:
Could be simplified further to `AWS.S3Access.X` or `AWS.S3.Access.X` possibly. This is also good.